lib/src/platform_text_field.dart

```diff
@@ -53,6 +53,54 @@ const BoxDecoration _kDefaultRoundedBorderDecoration = BoxDecoration(
   borderRadius: BorderRadius.all(Radius.circular(5.0)),
 );
 
+InputDecoration _inputDecorationWithHint(String hint, InputDecoration inputDecoration) {
+  return InputDecoration(
+    alignLabelWithHint: inputDecoration?.alignLabelWithHint ?? null,
```

Comment:
```suggestion
    alignLabelWithHint: inputDecoration?.alignLabelWithHint,
```

No need to do `?? null` since it will always be null if it gets that far. Same for all the `?? null` properties below. Just remove them